src/components/Maps/Marker.tsx

```diff
@@ -22,25 +34,36 @@ const Marker = ({
   lng,
   /* eslint-enable */
   isCurrentLocation,
-  otherClicked,
+  type,
+  clicked,
   onClick,
 }: {
   lat: number;
   lng: number;
   isCurrentLocation?: boolean;
-  otherClicked?: boolean;
+  type?: PinType;
+  clicked?: boolean;
   onClick?: () => void;
 }) => {
   const styles = useStyles();
   if (isCurrentLocation) {
     return <RadioButtonCheckedIcon className={styles.currentLocation} />;
   }
   return (
-    <RoomIcon
-      onClick={onClick}
-      fontSize="large"
-      className={otherClicked ? styles.notClicked : styles.root}
-    />
+    <Icon onClick={onClick} fontSize="large">
+      <img
+        alt="pin"
+        src={
+          clicked
+            ? currentSelectPin
+            : type === PinType.EVENT
+            ? eventPin
+            : type === PinType.CCP
+            ? ccpPin
+            : otherPin
+        }
```

Comment:
one way to make this a lot more readable is to check the `pinType` first, then check if it's clicked

e.g.
```
const pinSrc = type === PinType.EVENT ? eventPin
             : type === PinType.CCP ? ccpPin
             : otherPin;
...
        src={isClicked ? selectedPin : pinSrc}
```